lightkurve/targetpixelfile.py

```diff
@@ -1002,6 +1003,7 @@ def from_fits_images(images, position=None, size=(10, 10), extension=None,
         tpf : KeplerTargetPixelFile
             A new Target Pixel File assembled from the images.
         """
+        from lightkurve.targetpixelfile import _get_pos_corrs
```

Comment:
This needs to be removed as the function is no longer there. I think that's why the build is crashing.